articles/storage/common/storage-samples-dotnet.md

```diff
@@ -14,6 +14,9 @@ ms.topic: sample
 
 The following table provides an overview of our samples repository and the scenarios covered in each sample. Click on the links to view the corresponding sample code in GitHub.
 
+> [!NOTE]
+> These samples use our .NET v11 library. This page will be updated to v12 soon. For v12 samples, refer to the file in the GitHub repository found [here](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs/samples).
```

Comment:
As a policy, we don't make promises in the documentation. So, we want to remove the sentence that says, "This page will be updated to v12 soon." Also, we don't make links that say "click [here]." The link text should describe where it leads. Please update as follows. Thanks!

> [!NOTE]
> These samples use the Azure Storage .NET v11 library. For v12 code, see [samples](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.Storage.Blobs/samples) in the GitHub repository.

Reply:
Done!